internal/cli/run/run.go

```diff
@@ -89,7 +89,7 @@ func init() {
 			log.WithError(err).Error("Failed to lookup the location of the probe")
 			return err
 		}
-		network, err = database.CreateNetwork(ctx.DB, ctx.Session.Location)
+		network, err = database.CreateNetwork(ctx.DB, ctx.Session)
```

Comment:
We don't expose `Location` anymore since v0.1.0. It may be `null` and should be a private field. Plus, we can obtain the same info in a safer way by using the `ctx.Session` API.